### ISSUE TITLE
About the number of sentences in EMNLP2017 WMT News Dataset

### ISSUE THREAD

**Issue by liyzcj** [weilinie/RelGAN#3]
Hi , Thank you for sharing the code. I have a question here :
I saw in your paper you said the EMNLP dataset contains 10000 sentences both training  and test data , but why in the repo the file emnlp_news.txt contains 270,000 sentences.

and how you choose  10000 sentences from it ? I didn't see this part in the code..

Thank you : ) !

**Comment by weilinie** [comment]
Sorry it is a typo in the paper, and actually we use the whole 270,000 sentences in the emnlp_news.txt as our training dataset. We will update the paper. Thanks for pointing it out!

**Comment by liyzcj** [comment]
Ok, thank you . And I got another question here:

In the discriminator, why the logits shape is [4096 ,1] ?  I think it should be [64 , 1] .

**Comment by weilinie** [comment]
The fact that logits shape is [4096, 1] is according to Eq. (8) in Section 2.3 where we use S=64 embedded representations and 64 minibatches.
